src/status_im/constants.cljs

```diff
@@ -263,10 +263,23 @@
 (def ^:const wallet-connect-metadata-icon
   "https://res.cloudinary.com/dhgck7ebz/image/upload/f_auto,c_limit,w_1080,q_auto/Brand/Logo%20Section/Mark/Mark_01")
 (def ^:const wallet-connect-metadata-url "https://status.app")
-(def ^:const optimism-crosschain-id "eip155:10")
-(def ^:const wallet-connect-supported-methods ["eth_sendTransaction" "personal_sign"])
+
+(def ^:const wallet-connect-personal-sign-method "personal_sign")
+(def ^:const wallet-connect-eth-sign-method "eth_sign")
+(def ^:const wallet-connect-eth-send-transaction-method "eth_sendTransaction")
+(def ^:const wallet-connect-eth-sign-transaction-method "eth_signTransaction")
+(def ^:const wallet-connect-eth-sign-typed-method "eth_signTypedData")
+(def ^:const wallet-connect-eth-sign-typed-v4-method "eth_signTypedData_v4")
+(def ^:const wallet-connect-supported-methods
+  [wallet-connect-personal-sign-method
```

Comment:
It's not strictly necessary, but it would be better if we used a set instead of a vector here because that clearly communicates we care about uniqueness and we don't care about order.